### PR TITLE
Not registering the sharded variants in solo by default

### PR DESCRIPTION
**Pull request by PiSoup**

### CODE REVIEW

File: .solo/data-coordinator-alpha.json [hunk 1]
```diff
@@ -3,5 +3,6 @@
   "image_name": "data-coordinator",
   "ports": [ 6020 ],
   "start_container": "bin/start_dc_docker_alpha.sh",
-  "start_process": "bin/start_dc_alpha.sh"
+  "start_process": "bin/start_dc_alpha.sh",
+  "register_by_default: false"
```
Comment (tkbrigham) [inline .solo/data-coordinator-alpha.json]: Needs to be `"register_by_default": false`

File: .solo/data-coordinator-bravo.json [hunk 2]
```diff
@@ -3,5 +3,6 @@
   "image_name": "data-coordinator",
   "ports": [ 6040 ],
   "start_container": "bin/start_dc_docker_bravo.sh",
-  "start_process": "bin/start_dc_bravo.sh"
+  "start_process": "bin/start_dc_bravo.sh",
+  "register_by_default: false"
```
Comment (tkbrigham) [inline .solo/data-coordinator-bravo.json]: ditto

File: .solo/data-coordinator-charlie.json [hunk 3]
```diff
@@ -3,5 +3,6 @@
   "image_name": "data-coordinator",
   "ports": [ 6060 ],
   "start_container": "bin/start_dc_docker_charlie.sh",
-  "start_process": "bin/start_dc_charlie.sh"
+  "start_process": "bin/start_dc_charlie.sh",
+  "register_by_default: false"
```
Comment (tkbrigham) [inline .solo/data-coordinator-charlie.json]: ditto

**Comment by smooshy** [comment]
can you do it for `secondary-watcher-pg` as well?

**Comment by PiSoup** [comment]
already open https://github.com/socrata-platform/soql-postgres-adapter/pull/273

**Pull request merged**